wpilib/wpilib/pidcontroller.py

```diff
@@ -9,12 +9,15 @@
 from collections import deque
 from itertools import islice
 import threading
+from typing import Optional, NoReturn, Union
```

Comment:
You do use pyflakes, right? Right??

Reply:
It's hard to just use Pyflakes because it'll mark things that I didn't put in, and then I'd have to sift through the results lest you and dustin asking why I removed something that had nothing to do with type hints

Reply:
Sounds like we need to automatically run pyflakes...